crates/viewer/re_view_spatial/src/ui_3d.rs

```diff
@@ -450,13 +456,25 @@ impl SpatialView3D {
             query.view_id,
         );
 
+        let view_context = self.view_context(ctx, query.view_id, state);
+
+        let speed = **eye_property
+            .component_or_fallback::<LinearSpeed>(&view_context, &EyeControls3D::descriptor_speed())
+            .unwrap_debug_or_log_error() // Should never fail.
+            .unwrap_or(1.0.into());
```

Comment:
a bit annoying; why do we have to handle that here and not for instance when using `get_mono_with_fallback`? Creates a very strange asymmetry of error handling expectations